js/initcontext.go

```diff
@@ -70,7 +70,9 @@ func NewInitContext(
 		moduleRegistry:    getJSModules(),
 		exportsCache:      make(map[string]goja.Value),
 		moduleVUImpl: &moduleVUImpl{
-			ctx:     context.Background(), // TODO: pass a real context here as well?
+			// TODO: pass a real context as we did for https://github.com/grafana/k6/pull/2800,
+			// also see https://github.com/grafana/k6/pull/2800
```

Comment:
also see https://github.com/grafana/k6/pull/2804

Reply:
:man_facepalming: :sweat_smile: :+1:

Reply:
Resolved by the [latest force-push](https://github.com/grafana/k6/compare/9c9e35f63ef955140aff6ac122867832a249c7d8..a7ad53fef43a3a7885df96e08afab88c83d24436)